app/i18n/en/sub.php

```diff
@@ -1,6 +1,10 @@
 <?php
 
 return array(
+	'api' => array(
+		'documentation' => 'Copy the following URL to consume it with an external tool.',
```

Comment:
Consume? Seems like a slightly odd word choice. Maybe just `use`?

* `for use in an external tool`
* `to use it with an external tool`